modules/administration-guide/examples/system-variables.adoc

```diff
@@ -676,7 +676,7 @@ Default::: `+true+`
 
 == `+CHE_INFRA_KUBERNETES_NAMESPACE_LABELS+`
 
-List of labels to find {orch-namespace} that are used for {prod-short} Workspaces. They are used to:  - find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.annotations`.  - actively label {orch-namespace} with any workspace.
+List of labels to find {orch-namespace} that are used for {prod-short} Workspaces. They are used to:  - find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.annotations`.  - actively label {orch-namespace} with any workspace. NOTE: It is strongly recommended not to change the value of this property because the {prod-short} operator relies on these labels and their precise values when reconciling DevWorkspaces. If this configuration is changed, the namespaces will not be automatically recognized by the {prod-short} operator as workspace namespaces unless manually labeled as such using the default labels and values. Additional labels on the namespace do not affect the functionality.
```

Comment:
```suggestion
List of labels to find {orch-namespace} that are used for {prod-short} Workspaces. They are used to:  - find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.annotations`.  - actively label {orch-namespace} with any workspace. NOTE: It is strongly recommended not to change the value of this property because the {prod-short} operator relies on these labels and their precise values when reconciling workspace resources. If this configuration is changed, the namespaces will not be automatically recognized by the {prod-short} operator as workspace namespaces unless manually labeled as such using the default labels and values. Additional labels on the namespace do not affect the functionality.
```